packages/web/src/components/result/addons/Pagination.js

```diff
@@ -163,6 +164,12 @@ export default function Pagination(props) {
 	);
 }
 
+function areEqual(prevProps, nextProps) {
+	return isEqual(prevProps, nextProps);
+}
+
+export default React.memo(Pagination, areEqual);
```

Comment:
React.memo is not supported by React version <= 16.6

Reply:
True. 
Should I change to `class` Component and use `PureComponent` ?

Reply:
@jyash97 Yes, you can try it out.
Btw what is the reason of extra re-rendering?

Reply:
The reason for re-rendering is due to redux store: 

- Promoted Results
- Loading
- Hits Update
- URLParams Update

We can't stop this re-rendering as this can cause some side effects.